acceptance/tests/pxp-module-puppet/run_puppet_result_failed.rb

```diff
@@ -0,0 +1,94 @@
+require 'pxp-agent/test_helper.rb'
+require 'puppet/acceptance/environment_utils'
+
+test_name 'C93063 - pxp-module-puppet run expect puppet failure' do
+  extend Puppet::Acceptance::EnvironmentUtils
+
+  app_type = test_file_name = File.basename(__FILE__, '.*')
+  tmp_environment   = mk_tmp_environment(app_type)
+  tmp_environment2  = mk_tmp_environment(app_type)
+
+  step 'On master, create new environments that will result in Failed' do
+    site_manifest = File.join(environmentpath, tmp_environment, 'manifests', 'site.pp')
+    create_remote_file(master, site_manifest, <<-SITEPP)
+node default {
+  fail('intended failure')
+}
+SITEPP
+    site_manifest2 = File.join(environmentpath, tmp_environment2, 'manifests', 'site.pp')
+    create_remote_file(master, site_manifest2, <<-SITEPP)
+node default {
+  exec{'/bin/false':}
```

Comment:
After we resolve the comment below about running the environment; it would be worth double checking that this still results in an equivalent failure on Windows (and I think some posix platforms will be /usr/bin/false) - I *think* it will; as Puppet will get to the point of catalog execution and consider this resource a failed resource